docs/framework/data/wcf/accessing-data-service-resources-wcf-data-services.md

```diff
@@ -70,26 +70,28 @@ http://services.odata.org/Northwind/Northwind.svc/Orders?$filter=not endswith(Sh
-|`$top`|Specifies the number of entities to include in the returned feed. The following example skips the first 10 customers and then returns the next 10:<br /><br /> `http://services.odata.org/Northwind/Northwind.svc/Customers?$skip=10&$top=10`<br /><br /> For more information, see [OData: Top System Query Option ($top)](https://go.microsoft.com/fwlink/?LinkId=186969).|  
-|`$skip`|Specifies the number of entities to skip before starting to return entities in the feed. The following example skips the first 10 customers and then returns the next 10:<br /><br /> `http://services.odata.org/Northwind/Northwind.svc/Customers?$skip=10&$top=10`<br /><br /> For more information, see [OData: Skip System Query Option ($skip)](https://go.microsoft.com/fwlink/?LinkId=186971).|  
-|`$filter`|Defines an expression that filters the entities returned in the feed based on specific criteria. This query option supports a set of logical comparison operators, arithmetic operators, and predefined query functions that are used to evaluate the filter expression. The following example returns all orders the postal codes of which do not end in 100:<br /><br /> `http://services.odata.org/Northwind/Northwind.svc/Orders?$filter=not endswith(ShipPostalCode,'100')`<br /><br /> For more information, see [OData: Filter System Query Option ($filter)](https://go.microsoft.com/fwlink/?LinkId=186972).|  
-|`$expand`|Specifies which related entities are returned by the query. Related entities are included as either a feed or an entry inline with the entity returned by the query. The following example returns the order for the customer 'ALFKI' along with the item details for each order:<br /><br /> `http://services.odata.org/Northwind/Northwind.svc/Customers('ALFKI')/Orders?$expand=Order_Details`<br /><br /> For more information, see [OData: Expand System Query Option ($expand)](https://go.microsoft.com/fwlink/?LinkId=186973).|  
-|`$select`|Specifies a projection that defines the properties of the entity are returned in the projection. By default, all properties of an entity are returned in a feed. The following query returns only three properties of the `Customer` entity:<br /><br /> `http://services.odata.org/Northwind/Northwind.svc/Customers?$select=CustomerID,CompanyName,City`<br /><br /> For more information, see [OData: Select System Query Option ($select)](https://go.microsoft.com/fwlink/?LinkID=186076).|  
-|`$inlinecount`|Requests that a count of the number of entities returned in the feed be included with the feed. For more information, see [OData: Inlinecount System Query Option ($inlinecount)](https://go.microsoft.com/fwlink/?LinkId=186975).|  
-  
+|`$orderby`|Defines a default sort order for entities in the returned feed. The following query orders the returned customers feed by county and city:<br /><br /> `http://services.odata.org/Northwind/Northwind.svc/Customers?$orderby=Country,City`<br /><br />|  
```

Comment:
@mairaw These *odata.org* links support HTTPS. Are they tracked by CATS?

#4504

Reply:
Not when they are not in a link format since here they're marked as code. Given that this link actually works, we could create a PR to review these. 

And change this to

```suggestion
|`$orderby`|Defines a default sort order for entities in the returned feed. The following query orders the returned customers feed by county and city:<br /><br /> <https://services.odata.org/Northwind/Northwind.svc/Customers?$orderby=Country,City><br /><br />|  
```

Reply:
Opened PR #14263 to fix this